pkg/dotnet_library_license.txt

```diff
@@ -0,0 +1,127 @@
+MICROSOFT SOFTWARE LICENSE TERMS
+
+
+MICROSOFT .NET LIBRARY 
+
+These license terms are an agreement between Microsoft Corporation (or based on where you live, one of its affiliates) and you. Please read them. They apply to the software named above, which includes the media on which you received it, if any. The terms also apply to any Microsoft
+
+�         updates,
```

Comment:
If we close this pull request, will the updated text file which was submitted get picked up?